projects/plugins/jetpack/modules/protect/blocked-login-page.php

```diff
@@ -10,15 +10,42 @@
  * is available, or during the login_head hook.
  *
  * Class will only be instanciated if Protect has detected a hard blocked IP address.
- *
- *
  */
 class Jetpack_Protect_Blocked_Login_Page {
 
-	private static $__instance = null;
+	/**
+	 * Instance of the class.
+	 *
+	 * @var Jetpack_Protect_Blocked_Login_Page
+	 */
+	private static $instance = null;
+
+	/**
+	 * Can send recovery emails.
+	 *
+	 * @var bool - defaults to true.
+	 */
 	public $can_send_recovery_emails;
+
+	/**
+	 * The IP address.
+	 *
+	 * @var string
+	 */
 	public $ip_address;
+
+	/**
+	 * Valid blocked user ID.
+	 *
+	 * @var int - the user ID.
+	 */
 	public $valid_blocked_user_id;
+
+	/**
+	 * The email address.
+	 *
+	 * @var int - string.
```

Comment:
If that's an email address, shouldn't it be string?

```suggestion
	 * @var string
```